proxy-node-shared/src/main/resources/uk/gov/ida/notification/views/saml-form.mustache

```diff
@@ -15,40 +15,81 @@
         }
     </script>
     <style type='text/css'>
-      body {
-        padding-top: 2em;
-        padding-left: 2em;
-        background-color: white;
-      }
-      .saml-form {
-        font-family: Arial, sans-serif;
-      }
-      .submitBtn {
-        background-color: #00823b;
-        color: #fff;
-        padding: 10px;
-        font-size: 1em;
-        line-height: 1.25;
-        border: none;
-        box-shadow: 0 2px 0 #003618;
-        cursor: pointer;
-      }
-      .submitBtn:hover, .submitBtn:focus {
-        background-color: #00692f;
-      }
+        body {
```

Comment:
Wouldn't it be easier just to link or import some CSS file that's already deployed somewhere else in Verify?

Reply:
@jakubmiarka are those stylesheets deployed somewhere?

Reply:
they are but please don't hot link them.. they are versioned so the path is not guaranteed to be static...

Reply:
I take back what I said about 'easier'. 'More correct', maybe, but eh 😂